tutorials/creating-interiors/texturing.md

```diff
@@ -0,0 +1,42 @@
+# Texturing
+
+Next, we need to add materials to the Drawable Models. With one of the Drawable Models selected, open the `Sollumz 
+Tools > Drawable > Shader Tools` panel. For this tutorial, I will use the "NORMAL" shader which allows us to specify
+a base color and a normal map. Search for the "NORMAL" shader and click `Create Shader Material`.
+<figure><img src="../../.gitbook/assets/create_interior_tutorial_building17.png" alt=""><figcaption><p></p></figcaption></figure>
+
+To add a texture open shader tool and find the normal one select your drawable and then pres Create Shader Material, and
+it should appear in the Material menu in this tutorial I make 1 set of textures for each room so each room have a roof,
+wall, and floor. In this sample I use the normal, but you can play around with the others they have some different
+attributes that can be useful
+<figure><img src="../../.gitbook/assets/create_interior_tutorial_building18.gif" alt=""><figcaption><p></p></figcaption></figure>
+
+With the newly added material selected, head to the shading tab and assign your textures.
+<figure><img src="../../.gitbook/assets/create_interior_tutorial_building19.gif" alt=""><figcaption><p></p></figcaption></figure>
```

Comment:
Here, could you instead mention to do it through the material properties panel? The shading nodes have lots of settings that have no effect in-game and tend to confuse beginners. I prefer to guide them towards the material properties panel that only shows what's used by the game.
![image](https://github.com/user-attachments/assets/fc4044c0-6ba5-49cc-90f6-2160d35d75ac)